oca/host.py

```diff
@@ -78,6 +78,13 @@ def allocate(client, hostname, im, vmm, tm, cluster_id=-1):
         return host_id
 
     def __init__(self, xml, client):
+        # remove 'vn_mad' attribute in OpenNebula >= 5
+        xml_types = Host.XML_TYPES
+        if client.one_version is None:
+            client.version()
+        if client.one_version.startswith('5.') and 'vn_mad' in xml_types:
```

Comment:
I just realized, this is not the smartest way to check the version, as it will break with version 6.